src/utils/types/zetkin.ts

```diff
@@ -517,9 +517,10 @@ export interface ZetkinEmail {
 }
 
 export type ZetkinEmailPostBody = Partial<
-  Omit<ZetkinEmail, 'id' | 'published' | 'organization' | 'target'>
+  Omit<ZetkinEmail, 'id' | 'frame' | 'published' | 'organization' | 'target'>
```

Comment:
I liked your suggestion of renaming "frame" to "theme", it is easier to understand I think. Maybe a separate PR for that renaming process though? Or it could be done in here I guess, but maybe a bit messy to mix the ideas.